src/test/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepositoryTest.java

```diff
@@ -437,4 +437,38 @@ public void startJobSkipsNullParameters() {
 
     assertThat(parameters, is(empty()));
   }
+
+  @Test
+  public void parameters_populated_from_environment() {
```

Comment:
calling it "...from_cause" would be more descriptive I think

Reply:
`...from_StashCause` to be more precise.